packages/orchestration/src/utils/address.js

```diff
@@ -104,40 +105,52 @@ export const getBech32Prefix = address => {
 };
 
 /**
- * Parses an account ID into a structured format following CAIP-10 standards.
+ * Parse an account ID into a structured format following CAIP-10 standards.
  *
- * @param {string} partialId CAIP-10 account ID or an unscoped on-chain address
+ * @param {AccountIdArg} partialId CAIP-10 account ID or an unscoped on-chain
+ *   address
  * @returns {{
- *       namespace: string;
- *       reference: string;
- *       accountAddress: string;
- *     }
- *   | {
- *       accountAddress: string;
- *     }}
+ *   namespace: string;
+ *   reference: string;
+ *   accountAddress: string;
+ * }}
  *   - The parsed account details.
  */
-export const parseAccountId = partialId => {
-  if (typeof partialId !== 'string' || !partialId.length) {
-    Fail`Empty accountId: ${q(partialId)}`;
+export const parseAccountIdArg = partialId => {
```

Comment:
"partial" case is no longer supported.
```suggestion
export const parseAccountIdArg = idArg => {
```

Reply:
done